arangod/V8Server/v8-vocbase.cpp

```diff
@@ -762,18 +762,10 @@ static void JS_ExecuteAqlJson(v8::FunctionCallbackInfo<v8::Value> const& args) {
     TRI_V8ToVPack(isolate, options, args[1], false);
   }
 
-  arangodb::aql::QueryId queryId;
-  if (ServerState::instance()->isCoordinator()) {
-    queryId =
-        vocbase.server().getFeature<ClusterFeature>().clusterInfo().uniqid();
-  } else {
-    queryId = TRI_NewServerSpecificTick();
-  }
-
   auto origin = transaction::OperationOriginAQL{"executing query from JSON"};
-  auto query = arangodb::aql::ClusterQuery::create(
-      queryId, transaction::V8Context::create(vocbase, origin, true),
-      aql::QueryOptions(options.slice()));
+  auto query = aql::Query::create(
+      transaction::V8Context::create(vocbase, origin, true), aql::QueryString{},
+      /*bindParameters*/ nullptr, aql::QueryOptions(options.slice()));
```

Comment:
Using a `ClusterQuery` instance here has always been awkward, and now finally has been rectified.
We also don't need a query id here, as the regular `Query` will happily generate one if needed.